frame/lending/src/tests.rs

```diff
@@ -517,7 +514,7 @@ fn borrow_repay() {
 		assert_ok!(Lending::deposit_collateral_internal(&market, &BOB, bob_balance));
 		assert_eq!(Tokens::balance(MockCurrencyId::USDT, &BOB), 0);
 
-		let borrow_asset_deposit = 10 * MockCurrencyId::BTC.unit::<Balance>();
```

Comment:
is new and old value same?

Reply:
not sure i got question, but in short - no units in lending protocol.